executor/api/rabbitmq/server_test.go

```diff
@@ -129,89 +118,6 @@ func TestRabbitMqServer(t *testing.T) {
 		assert.Equal(t, fullHealthCheck, server.FullHealthCheck)
 	})
 
-	/*
-	 * This makes sure the Serve() and predictAndPublishResponse() code runs and makes the proper calls
-	 * by hacking a bunch of mocks.
-	 * It is not doing anything to validate the messages are properly processed.  That's challenging in a
-	 * unit test since the code connects to RabbitMQ.
-	 */
-	t.Run("serve", func(t *testing.T) {
```

Comment:
This is one of the main tests for the server, it should be updated to work with the changes and not removed